src/jit/flowgraph.cpp

```diff
@@ -23870,6 +23888,253 @@ void Compiler::fgUpdateFinallyTargetFlags()
             }
         }
     }
-
 #endif // FEATURE_EH_FUNCLETS && defined(_TARGET_ARM_)
 }
+
+class FatCalliTransformer
```

Comment:
In particular, I want to see comments that answer the questions: (1) what is a fat pointer? (2) what does one look like? (3) where do they come from? (4) how are they used? (5) what platforms use them, and why? (6) what does the JIT need to do to make them work? (7) what does the generated code look like; show an example.

---
In reply to: [99206797](https://github.com/dotnet/coreclr/pull/9191#discussion_r99206797) [](ancestors = 99206797)